### PR TITLE
Update UG for delete and find commands

### PR DESCRIPTION
**Pull request by haleemairfan**

### CODE REVIEW

**Review by MinkhantEain — APPROVED** [review]
LGTM.

**Review by yyueyii — CHANGES_REQUESTED** [review]
Some minor errors, otherwise lgtm!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -157,33 +183,57 @@ Examples:
...
+### Phone Number (`p/PHONE`)
+- Deletes the person with the specified **full** `PHONE_NUMBER`.
+- **Exact Match Required**: Partial phone numbers will not be matched.
+- **Unique Match**: Since phone numbers are unique, Cher will directly delete the person with that phone number.
 
-Examples:
-* `list` followed by `delete John` deletes the person with the name John in the address book.
-* If multiple persons have the name John, a list of these persons is displayed to the user
+### Examples
+- `list` followed by `delete 98765432` deletes the person with the phone number `98765432` in the list.
```
Comment (yyueyii) [inline docs/UserGuide.md:220]: `delete p/98765432`

**Review by yyueyii — APPROVED** [review]
LGTM!

**Pull request merged**